testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -312,11 +322,32 @@ fn load_tc_key(tc_key_bytes: &[u8]) -> Ed25519PrivateKey {
     Ed25519PrivateKey::try_from(tc_key_bytes).unwrap()
 }
 
+async fn wait_genesis_job(kube_client: &K8sClient, era: usize) -> Result<(), anyhow::Error> {
+    diem_retrier::retry_async(k8s_retry_strategy(), || {
+        let jobs: Api<Job> = Api::namespaced(kube_client.clone(), "default");
+        Box::pin(async move {
+            let job_name = format!("diem-testnet-genesis-e{}", era);
+            println!("Running get job: {}", &job_name);
+            let genesis_job = jobs.get_status(&job_name).await.unwrap();
+            println!("Status: {:?}", genesis_job.status);
+            let status = genesis_job.status.unwrap();
+            match status.succeeded {
+                Some(1) => {
+                    println!("Genesis job completed");
+                    Ok(())
+                }
+                _ => bail!("Genesis job not completed"),
+            }
+        })
+    })
+    .await
+}
+
 pub fn clean_k8s_cluster(
     helm_repo: String,
     base_num_validators: usize,
```

Comment:
what if we change this to total_num_validators? if I understand correctly this number stands for all validators are running on cluster?

Reply:
I was trying to disambiguate it from the total number of validators that are *currently* running in the cluster. That variable is meant to be the number of validators running in the cluster's base/stable state.